test/gtest/ucp/test_ucp_proto.cc

```diff
@@ -431,16 +441,118 @@ class test_proto_perf : public ucs::test {
         return ucp_proto_perf_find_segment_lb(perf.get(), start);
     }
 
-    void expect_empty_range(size_t start, size_t end)
+    static const ucp_proto_flat_perf_range_t *
+    find_lb(flat_perf_ptr_t flat_perf, size_t start)
     {
-        ucp_proto_perf_segment_t *seg = find_lb(m_perf, start);
+        return ucp_proto_flat_perf_find_lb(flat_perf.get(), start);
+    }
+
+    static size_t get_start(ucp_proto_perf_segment_t *seg) {
```

Comment:
const

Reply:
Let's discuss that according to the previous comment about const.